app/javascript/react/components/ReviewFormContainer.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from "react"
+
+const ReviewTileContainer = (props) => {
```

Comment:
big uh oh. Your component is called ReviewTile, but the file is called ReviewFormContainer? lets reconcile this

Reply:
Deleted ReviewFormContainer